fedmsg.d/endpoints.py

```diff
@@ -29,15 +29,15 @@
         # infrastructure's bus.  Cool, right?  :)
         "fedora-infrastructure": [
             "tcp://hub.fedoraproject.org:9940",
-            #"tcp://stg.fedoraproject.org:9940",
+            # "tcp://stg.fedoraproject.org:9940",
         ],
 
         # For other, more 'normal' services, fedmsg will try to guess the
         # name of it's calling module to determine which endpoint definition
         # to use.  This can be overridden by explicitly providing the name in
         # the initial call to fedmsg.init(...).
-        #"bodhi.%s" % hostname: ["tcp://127.0.0.1:3001"],
-        #"fas.%s" % hostname: ["tcp://127.0.0.1:3002"],
-        #"fedoratagger.%s" % hostname: ["tcp://127.0.0.1:3003"],
+        # "bodhi.%s" % hostname: ["tcp://127.0.0.1:3001"],
+        # "fas.%s" % hostname: ["tcp://127.0.0.1:3002"],
+        # "fedoratagger.%s" % hostname: ["tcp://127.0.0.1:3003"],
```

Comment:
Yeah, I've seen this go both ways. My personal preference is to have the space always, but that might be because I'm used to ``Ctrl+v`` in vim. In this case because it's inside a dictionary literal, indentation doesn't matter so you can still get rid of the ``#`` and everything will Just Work.